Integrations/PaloAltoNetworksCortex/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Fixed an issue with fetch incidents.
```

Comment:
```suggestion
Fixed an issue with fetch incidents in which failed job raised an exception.
```

Reply:
Fixed an issue with the fetch incidents function in which failed jobs raised an exception.